tardis/model/geometry/tests/test_radial1d.py

```diff
@@ -0,0 +1,79 @@
+from astropy import units as u
+import numpy as np
+
+from tardis.model.geometry.radial1d import HomologousRadial1DGeometry
+
+import pytest
+
+
+@pytest.fixture(scope="function")
+def homologous_radial1d_geometry():
+    velocity = np.arange(8000, 21000, 1000) * u.km / u.s
+    v_inner = velocity[:-1]
+    v_outer = velocity[1:]
+    time_explosion = 5 * u.day
+    geometry = HomologousRadial1DGeometry(
+        v_inner, v_outer, v_inner[0], v_outer[-1], time_explosion
+    )
+    return geometry
+
+
+def test_v_indices(homologous_radial1d_geometry):
+    # Testing if the indices returned are correct when inner and outer
+    # boundary are on the innermost and outermost shell
+
+    homologous_radial1d_geometry.v_inner_boundary = (
+        homologous_radial1d_geometry.v_inner[0]
+    )
+    homologous_radial1d_geometry.v_outer_boundary = (
+        homologous_radial1d_geometry.v_outer[-1]
+    )
+    assert homologous_radial1d_geometry.v_inner_boundary_index == 0
+    assert homologous_radial1d_geometry.v_outer_boundary_index == 12
+    vib_index = homologous_radial1d_geometry.v_inner_boundary_index
+    vob_index = homologous_radial1d_geometry.v_outer_boundary_index
+    assert np.all(
+        homologous_radial1d_geometry.v_inner[vib_index:vob_index]
+        == homologous_radial1d_geometry.v_inner
+    )
+
+    # pivoting around the inner boundary of the simulation
```

Comment:
@andrewfullard @Rodot- - can you see if that logic is correct

Reply:
Makes sense to me, though the logic here shows that your implementation does not report any error if something is "inside" the inner boundary or "outside" the outer boundary. Is that intended?

Reply:
that is an easy fix - for now. I would just think that we keep things the same. So if we have 8000 - 9000 - density = 1e-13 - then v_inner_boundary at 7500, then the new cell will be 7500 - 9000 with density 1e-13.

Reply:
Okay. I assume this does *not* affect packets going within the inner boundary or leaving the outer boundary then?

Reply:
nope